### PR TITLE
docs(rxCollapse): FRMW-170-Migrate rxCollapse to API docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-170

- [x] Dev LGTM
- [x] Dev LGTM

Migrate rxCollapse to API documentation.

### CODE REVIEW

File: src/rxCollapse/rxCollapse.js [hunk 1]
```diff
@@ -18,6 +20,18 @@ angular.module('encore.ui.rxCollapse', [])
  * @description
  * Hide and show an element with a transition.
  *
+ * ## Hide/Show Content
+ * 
+ * - This pattern was developed for areas displaying metadata that may be short on screen real estate, as a way to hide
```
Comment (bry4n) [inline src/rxCollapse/rxCollapse.js]: replace `-` with `*`

**Comment by 100stacks** [comment]
:+1:

**Comment by 100stacks** [comment]
screen shot 2015-09-28 at 10 53 01 am

File: src/rxCollapse/README.md [hunk 2]
```diff
@@ -1,15 +1,3 @@
...
-- This pattern was developed for areas displaying metadata that may be short on screen real estate, as a way to hide data on load that is not as important to the user in the context where they are presented.
-- This pattern is not very responsive-friendly, since as browser width decreases, columns will wrap. As columns wrap, the "See More" rxCollapse elements get lost in the new context, which is bad for user experience.
-- To avoid the problem described above, "See More" rxCollapse elements should only be used at the end of the final column present on the page, so that when the columns wrap via flexbox, "See More" is always last and doesn't get lost in between metadata key/value pairs.
-
-
-### Attributes
-
-- `title`: The title to display next to the toggle button. Remove or leave the title blank in order to enable "See More / See Less" version of rxCollapse.
-- `expanded`: Initially expanded or collapsed.  Default is expanded.
+`rxCollapse` element can be used to display and hide content.  A double chevron(**>>**) is used to toggle between showing and hiding contents.
```
Comment (CITguy) [inline src/rxCollapse/README.md]: This wording is specific to the directive and not the component. Please update wording to be for component description.

File: src/rxCollapse/rxCollapse.js [hunk 3]
```diff
@@ -4,7 +4,9 @@
  * @description
  * # rxCollapse Component
  *
- * [TBD]
+ * `rxCollapse` element can be used to display and hide content. It can be configured to show as either expanded
```
Comment (CITguy) [inline src/rxCollapse/rxCollapse.js]: Same as above comment.

**Comment by 100stacks** [comment]
:ok_hand: :+1:

File: src/rxCollapse/rxCollapse.js [hunk 4]
```diff
@@ -18,6 +20,18 @@ angular.module('encore.ui.rxCollapse', [])
...
+ * * This pattern is not very responsive-friendly, since as browser width decreases, columns will wrap. As columns wrap,
+ *  the "*See More*" `rxCollapse` elements get lost in the new context, which is bad for user experience.
+ * * To avoid the problem described above, "*See More*" `rxCollapse` elements should only be used at the end of the
+ * final column present on the page, so that when the columns wrap via flexbox, "*See More*" is always last and doesn't 
+ * get lost in between metadata key/value pairs.
+ *
+ * 
  * @param {String=} [title="See More/See Less"]
  * The title to display next to the toggle button. Default is "See More/See Less" toggle.
  * @param {String=} [expanded='true']
```
Comment (100stacks) [inline src/rxCollapse/rxCollapse.js]: @CITguy should `@param` `expanded` be `boolean`?  The current type is `String`

Comment (CITguy) [inline src/rxCollapse/rxCollapse.js]: Yes, `Boolean`

```
@param {Boolean=} [expanded=true] ...
```

Comment (100stacks) [inline src/rxCollapse/rxCollapse.js]: Okay, fixed :ok_hand:

**Comment by CITguy** [comment]
LGTM

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
@100stacks, please squash PR

**Pull request merged**